src/libraries/System.Diagnostics.Process/src/System.Diagnostics.Process.csproj

```diff
@@ -10,6 +10,9 @@
   <PropertyGroup>
     <GeneratePlatformNotSupportedAssemblyMessage Condition="'$(TargetsAnyOS)' == 'true'">SR.Process_PlatformNotSupported</GeneratePlatformNotSupportedAssemblyMessage>
   </PropertyGroup>
+   <PropertyGroup Condition="'$(Platform)' == 'mips64'">
```

Comment:
src/libraries csharp files are built as architecture-neutral. Switching on architecture here is not going to work well with the rest of the build.

The better way to fix this is to remap the signal numbers under https://github.com/dotnet/runtime/tree/master/src/libraries/Native